extensions/binding/org.eclipse.smarthome.binding.lifx/src/main/java/org/eclipse/smarthome/binding/lifx/internal/protocol/SetColorRequest.java

```diff
@@ -68,13 +51,9 @@ public SetColorRequest() {
         setResponseRequired(true);
     }
 
-    public SetColorRequest(int hue, int saturation, int brightness, int kelvin, long fadeTime) {
+    public SetColorRequest(HSBK color, long fadeTime) {
         this();
```

Comment:
Not your change, but as commented above, I don't think we need to call the default constructor explicit all the time.